Makefile

```diff
@@ -1,5 +1,16 @@
 # Image URL to use all building/pushing image targets
-IMG ?= controller:latest
+IMAGE_REGISTRY ?= quay.io
+REGISTRY_NAMESPACE ?= ocs-dev
+IMAGE_TAG ?= latest
+IMAGE_NAME ?= ceph-csi-operator
```

Comment:
We will need these changes in the upstream as well

Reply:
can you please send a PR for this change in upstream?

Reply:
Here is a PR https://github.com/ceph/ceph-csi-operator/pull/30